app/controllers/application_controller.rb

```diff
@@ -39,20 +39,32 @@ def catch_exceptions
            status: :bad_request
   end
 
-  def catch_flexirest_exceptions
+  def catch_flexirest_exceptions # rubocop:disable Metrics/AbcSize
     yield
   rescue Flexirest::HTTPClientException,
          Flexirest::HTTPServerException,
          Flexirest::HTTPNotFoundClientException => e
 
-    errors_hash = e.result.Errors.to_a.map do |err|
-      { "title": err.to_hash['Message'] }
-    end
+    errors_hash = if e.result.respond_to?(:Errors)
```

Comment:
I'd go ahead and just extract this into a method. It'll fix the formatting, and make it clearer, and (I think) remove the need to disable the lint warning.

Reply:
Thanks for the suggestion - moved out to a method (is definitely cleaner) - but still had the lint warning